CONTRIBUTING.md

```diff
@@ -69,3 +69,87 @@ header.
 6. Create a pull request against the branch `master`.
 7. Be prepared to get some comments about your code and to modify it. Tip: Use
    `git rebase -i origin/master` to modify chains of commits.
+
+## Adding a test
+
+The test suite is a little ad-hoc, so the way tests work is different for each program being tested.
+
+Feel free to modify how the test scripts work, 
+
+### RGBASM
+
+Each `.asm` file corresponds to one test.
+RGBASM will be invoked on the `.asm` file.
+
+If a `.out` file exists, RGBASM's output (`println` etc.) must match its contents.
+If a `.err` file exists, RGBASM's errors (`warn`, errors, etc.) must match its contents.
+
+If a `.out.bin` file exists, the object file will be linked, and the generated ROM truncated to the length of the `.out.bin` file.
+After that, the ROM must match the `.out.bin` file.
+
+### RGBLINK
+
+Each `.asm` file corresponds to one test, or one *set* of tests.
+
+All tests begin by assembling the `.asm` file into an object file, which will be linked in various ways depending on the test.
+
```

Comment:
Likewise for how to make the expected output of RGBLINK tests (they do have `.out` and `.out.bin` files).

Reply:
See https://github.com/gbdev/rgbds/pull/1324/files#r1509967319